### PR TITLE
revise: makes various improvements to the `line_width` rule

### PR DESCRIPTION
**Pull request by claymcleod**
In attempting to fix the `line_width` rule, I discovered that the problem was actually in the way we did excepting of rules. That's probably a PR for a different day, but I made some improvements to the rule in terms of conciseness that I'd like to go ahead and merge in.

### CODE REVIEW

File: Arena.toml [hunk 1]
```diff
@@ -1115,787 +1115,82 @@ document = "getwilds/ww-vc-trio:/ww-vc-trio.wdl"
 message = "ww-vc-trio.wdl:98:29: note[CallInputSpacing]: call input not properly spaced"
 permalink = "https://github.com/getwilds/ww-vc-trio/blob/c2c13e85efda8ac7aca7f8765fbaa7c2cacb08b2/ww-vc-trio.wdl/#L98"
 
-[[diagnostics]]
```
Comment (claymcleod) [inline Arena.toml:1118]: All of these are because the workflows repo updated and @a-frantz resolved most of the lints.

File: Arena.toml [hunk 2]
```diff
@@ -1115,787 +1115,82 @@ document = "getwilds/ww-vc-trio:/ww-vc-trio.wdl"
...
-document = "stjudecloud/workflows:/data_structures/read_group.wdl"
-message = "read_group.wdl:8:1: note[LineWidth]: line exceeds maximum width of 90"
-permalink = "https://github.com/stjudecloud/workflows/blob/833b97fb582fe7d4f6df9d29a645d0289f1fc92e/data_structures/read_group.wdl/#L8"
-
-[[diagnostics]]
-document = "stjudecloud/workflows:/tools/arriba.wdl"
-message = "arriba.wdl:140:1: note[LineWidth]: line exceeds maximum width of 90"
-permalink = "https://github.com/stjudecloud/workflows/blob/833b97fb582fe7d4f6df9d29a645d0289f1fc92e/tools/arriba.wdl/#L140"
+message = "read_group.wdl:134:1: note[LineWidth]: line exceeds maximum width of 90"
+permalink = "https://github.com/stjudecloud/workflows/blob/129060215443bbd1b67c4d3d149d48acdab90ef2/data_structures/read_group.wdl/#L134"
```
Comment (claymcleod) [inline Arena.toml:1121]: This is still going to exist until we implement the fix we discussed in Slack. That is probably the another PR in the future.

File: Arena.toml [hunk 3]
```diff
@@ -1115,787 +1115,82 @@ document = "getwilds/ww-vc-trio:/ww-vc-trio.wdl"
...
-document = "stjudecloud/workflows:/tools/cellranger.wdl"
-message = "cellranger.wdl:78:1: note[LineWidth]: line exceeds maximum width of 90"
-permalink = "https://github.com/stjudecloud/workflows/blob/833b97fb582fe7d4f6df9d29a645d0289f1fc92e/tools/cellranger.wdl/#L78"
-
-[[diagnostics]]
-document = "stjudecloud/workflows:/tools/deeptools.wdl"
-message = "deeptools.wdl:6:7: note[Todo]: remaining `TODO` item found"
-permalink = "https://github.com/stjudecloud/workflows/blob/833b97fb582fe7d4f6df9d29a645d0289f1fc92e/tools/deeptools.wdl/#L6"
+message = "arriba.wdl:147:1: note[LineWidth]: line exceeds maximum width of 90"
+permalink = "https://github.com/stjudecloud/workflows/blob/129060215443bbd1b67c4d3d149d48acdab90ef2/tools/arriba.wdl/#L147"
```
Comment (claymcleod) [inline Arena.toml:1126]: Same here.

File: wdl-lint/src/rules/line_width.rs [hunk 4]
```diff
@@ -89,8 +83,7 @@ impl Rule for LineWidthRule {
     }
 
     fn description(&self) -> &'static str {
-        "Ensures that lines do not exceed a certain width. That width is currently set to 90 \
-         characters."
+        "Ensures that lines do not exceed a certain width."
```
Comment (claymcleod) [inline wdl-lint/src/rules/line_width.rs:86]: The default should be described in the explanation (which it is), so it can be removed here.

**Comment by claymcleod** [comment]
Explicitly, I don't think this needs a CHANGELOG.md entry, as just the internals of the rule changed (and not it's behavior or interface).

File: wdl-lint/tests/lints/line-width/source.errors [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+   │
+14 │             this line is also going to be very very very long just to trip up our maximum line lint /
+   │ ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+   │
+   = fix: split the line into multiple lines
+
+note[LineWidth]: line exceeds maximum width of 90
+   ┌─ tests/lints/line-width/source.wdl:31:1
+   │  
+31 │ ╭     command <<<
```
Comment (claymcleod) [inline wdl-lint/tests/lints/line-width/source.errors:20]: See the comment in the source text for this diagnostic.

**Review by a-frantz — APPROVED** [review]

**Pull request merged**